.browserslistrc

```diff
@@ -5,3 +5,4 @@ last 1 Edge version
 last 1 Firefox version
 last 1 iOS version
 last 1 Safari version
+IE 10
```

Comment:
@somewhatabstract this is the change that forces ES5 output.  `@babel/preset-env` looks at this config file and uses it to decide which syntax constructs to transform or not.  The old version of this list transformed barely anything, adding IE 10 should guarantee that we'll be able to also run in phantomjs.  I also checked the output in the dist/ folders for any uses of `class` or `=>` and there weren't any in the output code.